specs/jsonschema-core.md

```diff
@@ -410,20 +410,37 @@ keywords MUST NOT begin with this prefix.
 Implementations MUST refuse to evaluate schemas which contain keywords which
 they do not know how to process or explicitly choose not to process.
 
-## Keyword Behaviors
+## Keyword Behaviors {#keyword-behaviors}
 
-JSON Schema keywords fall into several general behavior categories. Assertions
-validate that an instance satisfies constraints, producing a boolean result.
-Annotations attach information that applications may use in any way they see
-fit. Applicators apply subschemas to parts of the instance and combine their
-results.
+JSON Schema keywords may exhibit one or more behaviors. This specification
+defines three such behaviors:
```

Comment:
This seems to say that all keyword behaviors in the specification are in these three categories, but the spec defines other behaviors for e.g. `$id` and `$anchor`/`$dynamicAnchor`. And `$schema` ... exists.

Reply:
That is an interesting point.  I was thinking that these keywords don't exhibit a _behavior_ so much as they are merely informative, like `maxContains` would be to `contains`.

Reply:
This is a great point! My implementation doesn't even treat those as keywords. They're meta-data used to process the schema, but they're not directly part of the validation/annotation process. So, I wouldn't consider them similar to `maxContains` that is involved in validation.

I'm not sure what I'd suggest, but I think something needs to be called that these aren't normal keywords.